testsuite/tests/libratest/smoke_test.rs

```diff
@@ -123,7 +123,7 @@ impl TestEnvironment {
             .to_string();
 
         ClientProxy::new(
-            "localhost",
+            "http://localhost",
```

Comment:
nit: looks a bit unusual. Maybe make http by default?

Reply:
I'm fine with this suggestion - @thefallentree thoughts?